### PR TITLE
Move Feature tooltip that guides the user towards Node Tool if s(h)e wants snapping

### PR DESCRIPTION
**Pull request by tudorbarascu**
…ing usage

## Description
Include a few sentences describing the overall goals for this PR (pull request). If applicable also add screenshots.

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

- [x] Commit messages are descriptive and explain the rationale for changes
- [x] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [ ] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [ ] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and containt sufficient information in the commit message to be documented
- [x] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] New unit tests have been added for core changes
- [x] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) before each commit

tooltip

Don't know if this is the best way to guide the user but I cannot think of something else.

For the current node tool (really nice job!), we can select whatever nodes we want to move from a feature (all the nodes if we want to move the whole feature) and benefit from snapping.

What's a little hard to do now is select just the nodes we want as the rectangle selection is restrictive. For example nodes from other features can be inside the described rectangle and they also get selected.
I tried a modifier (Ctrl or Shift) to Add/Remove from the selection the wanted/unwanted nodes but it seems something like this is not implemented.

Suggestions?

### CODE REVIEW

File: src/ui/qgisapp.ui [hunk 1]
```diff
@@ -852,6 +852,9 @@
    <property name="text">
     <string>Move Feature(s)</string>
    </property>
+   <property name="toolTip">
+    <string>Move Feature(s); to benefit from snapping use the Node Tool</string>
```
Comment (m-kuhn) [inline src/ui/qgisapp.ui]: Might be something like

```
<b>Move Feature(s)</b>
<p>Try the <i>Node Tool</i> if you need snapping.</p>
```

**Comment by tudorbarascu** [comment]
@m-kuhn Good suggestion! I've rebased and pushed.
tooltip

I couldn't do the bold/unbold part as every text is with bold by default (except the shortcut parts - but it didn't feel right to put the tip there).

**Comment by m-kuhn** [comment]
Wondering, do we have some general HIG on addressing the user directly (i.e. the usage of the term "you")?
CC @DelazJ

**Comment by DelazJ** [comment]
@m-kuhn None that I'm aware of and not sure we often use this kind of direct addressing. Maybe `Try the Node Tool if snapping is needed`?

**Pull request merged**